frontend/src/main/components/OurTable.js

```diff
@@ -4,9 +4,9 @@ import { Table, Button } from "react-bootstrap";
 import Plaintext from "main/components/Utils/Plaintext";
 
 // Stryker disable all
-var tableStyle = {
-  "background": "white"
-};
+// var tableStyle = {
+//   "background": "white"
+// };
```

Comment:
Why are we commenting out these three lines?

This is undoing someone else's work.  Please don't do that.